common/containers/Tabs/SignAndVerifyMessage/components/SignMessage/index.tsx

```diff
@@ -10,7 +10,7 @@ import { AppState } from 'reducers';
 import './index.scss';
 
 interface Props {
-  wallet: IWallet;
+  wallet: IFullWallet;
```

Comment:
@HenryNguyen5 Another instance of specifying `IFullWallet` but typescript allowing any `IWallet`.

Reply:
Come to think of it, the reducer doesn't even say it's a sure thing; it's `wallet?: IWallet | null`. Kinda scares me that it's letting this thru.